modules/catalog/src/main/java/org/apache/ignite/internal/catalog/Catalog.java

```diff
@@ -115,14 +116,22 @@ public Collection<CatalogSchemaDescriptor> schemas() {
         return schemasByName.values();
     }
 
-    public CatalogTableDescriptor table(int tableId) {
+    public @Nullable CatalogTableDescriptor table(int tableId) {
```

Comment:
Why are there no javadocs in this class?

Reply:
I'm not the author of this class, I don't know why there is no documentation, can I create a ticket to add documentation or should I add documentation for my methods or for the entire class?

Reply:
Let's do it in a separate ticket

Reply:
Created ticket https://issues.apache.org/jira/browse/IGNITE-20046